source/crud/tests/unified/bulkWrite-arrayFilters-clientError.yml

```diff
@@ -0,0 +1,101 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: bulkWrite-arrayFilters-clientError
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name crud-v2
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name crud-v2
+runOnRequirements:
+  -
+    maxServerVersion: 3.5.5
+initialData:
+  -
+    collectionName: *collection_name
+    databaseName: *database_name
+    documents:
+      -
+        _id: 1
+        'y':
```

Comment:
Why is the "y" key quoted, but not "_id" above? I would expect these to be consistent if they were both produced by the converter script. Previous files didn't quote fields (e.g. `x`) in `intialData.documents` either.

Reply:
y is a shorthand for true in some parsers, hence the quotes applied by symfony/yaml